<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>705</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>705</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Suivi des prescriptions</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidgetPrincipale">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>671</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabRenouv">
     <attribute name="title">
      <string>A renouveler</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxRenouv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>A renouveler rapidement</string>
      </property>
      <widget class="QTableView" name="tableViewRenouv">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxDetails">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Détails</string>
      </property>
      <widget class="QLabel" name="labelPatient">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Patient :</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPrescription">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prescription :</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMedecin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Médecin :</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPatientValue">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelMedecinValue">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>216</width>
         <height>106</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelPresValable">
            <property name="text">
             <string>Valable jusqu'au :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPresDuree">
            <property name="text">
             <string>Durée de la prescription :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPresType">
            <property name="text">
             <string>Type de prescription :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPresModeRenouv">
            <property name="text">
             <string>Mode de renouvellement :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelPresValableValue">
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPresDureeValue">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPresTypeValue">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPresModeRenouvValue">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxActions">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>290</y>
        <width>241</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Action</string>
      </property>
      <widget class="QDateEdit" name="dateEditRenouveller">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRenouveller">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Renouveller</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPrescri">
     <attribute name="title">
      <string>Prescriptions</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabPatients">
     <attribute name="title">
      <string>Patients</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabMed">
     <attribute name="title">
      <string>Médecins</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabPara">
     <attribute name="title">
      <string>Paramètres</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
